spec/serializer_spec.rb

```diff
@@ -460,21 +485,21 @@ def serialize_primary(object, options = {})
       expect(actual_data['included']).to eq(expected_data['included'])
       expect(actual_data).to eq(expected_data)
     end
-    it 'includes linkage in compounded resources only if the immediate parent was also included' do
+    it 'includes data in compounded resources only if the immediate parent was also included' do
```

Comment:
Actually can you leave this as "linkage" instead of "data" to be less ambiguous? The spec still refers to these as 
"linkage" objects even though the actual thing uses a "data" key.